src/components/header/HelpButton.jsx

```diff
@@ -15,7 +15,16 @@ const HelpButton = () => {
     <Card size='small' style={{ padding: '1em', width: '265px' }}>
       {accountId !== AccountId.HMS && (
         <>
-          For tutorial videos, ‘how to’ guides and FAQs on how to use Cellenics, visit
+          The
+          {' '}
```

Comment:
this link can't be in the hms instance -- can you show it only for the community instance?

Reply:
It is already within the accountId !== HMS check

Reply:
yes, the link wouldn't show. For HMS instance, only the support email link is shown.